### ISSUE TITLE
sklweka import error

### ISSUE THREAD

**Issue by dmohsen23** [fracpete/scikit-weka#7]
Hello there
I have installed PyPI on my environment with three different methods that have been presented in the Installation guideline, and I receive "Requirement already satisfied" message when I try to reinstall it.
I have already created an environment for WEKA and I can use it; however, sklweka function does not work and I can't find the issue.

I would much appreciate it if you can help me to solve this problem.

**Comment by fracpete** [comment]
What OS are you on?
What Python version are you using?
Are you using a virtual environment?
What is the output of `pip freeze` in that environment?
How do you execute your script (IDE or command-line)?

**Comment by dmohsen23** [comment]
I use win64, and I use python 3.9.7
I have created a virtual environment in which WEKA is runnable on it. I normally use Jupiter notebook with VS Code but I have run the script on a python file as well but couldn't have a successful run!
These are my installed packages on python:

absl-py==0.15.0
aiohttp==3.8.3
aiosignal==1.2.0
anyio==3.6.2
arviz==0.12.1
astor==0.8.1
astunparse==1.6.3
async-timeout==4.0.2
asynctest==0.13.0
attrs==22.1.0
autograd==1.4
autograd-gamma==0.5.0
backcall==0.2.0
bayesian-optimization==1.2.0
bcrypt==4.0.1
cached-property==1.5.2
cachetools==5.2.0
certifi==2022.6.15
cffi==1.15.1
cftime==1.6.2
charset-normalizer==2.1.0
click==8.0.4
cloudpickle==2.2.0
colorama==0.4.5
coloredlogs==15.0.1
contourpy==1.0.5
cryptography==38.0.1
cycler==0.11.0
debugpy==1.6.2
decorator==5.1.1
deprecat==2.1.1
Deprecated==1.2.13
dill==0.3.5.1
distlib==0.3.6
entrypoints==0.4
fastapi==0.85.1
fastprogress==1.0.3
ffmpy==0.3.0
filelock==3.8.0
flatbuffers==1.12
FLORIS==2.1.1
fonttools==4.37.4
formulaic==0.2.4
frozenlist==1.3.1
fsspec==2022.10.0
fst-pso==1.8.1
future==0.18.2
gast==0.4.0
google-auth==2.9.1
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
gradio==3.8.1
grpcio==1.34.1
gurobipy==9.1.2
h11==0.12.0
h5py==3.1.0
httpcore==0.15.0
httpx==0.23.0
humanfriendly==10.0
hyperopt==0.2.7
idna==3.3
imbalanced-learn==0.9.0
imblearn==0.0
importlib-metadata==4.12.0
importlib-resources==5.10.0
iniconfig==1.1.1
intel-openmp==2022.2.0
interface-meta==1.3.0
ipykernel==6.15.1
ipython==7.34.0
javabridge==1.0.19
jedi==0.18.1
Jinja2==3.1.2
joblib==1.2.0
jsonschema==4.17.0
jupyter-client==7.3.4
jupyter-core==4.11.1
keras-nightly==2.5.0.dev2021032900
Keras-Preprocessing==1.1.2
kiwisolver==1.4.4
lifelines==0.27.1
lightgbm==3.3.3
linkify-it-py==1.0.3
Markdown==3.4.1
markdown-it-py==2.1.0
MarkupSafe==2.1.1
matplotlib==3.5.2
matplotlib-inline==0.1.3
mdit-py-plugins==0.3.1
mdurl==0.1.2
miniful==0.0.6
mkl==2022.2.0
msgpack==1.0.4
multidict==6.0.2
nest-asyncio==1.5.5
netCDF4==1.6.1
networkx==2.6.3
numpy==1.21.6
oauthlib==3.2.0
opt-einsum==3.3.0
orjson==3.8.1
packaging==21.3
pandas==1.1.5
paramiko==2.11.0
parso==0.8.3
patsy==0.5.2
pickleshare==0.7.5
Pillow==9.2.0
pkgutil_resolve_name==1.3.10
platformdirs==2.5.3
pluggy==1.0.0
progressbar2==4.0.0
prompt-toolkit==3.0.30
protobuf==3.19.4
psutil==5.9.1
py==1.11.0
py4j==0.10.9.7
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycparser==2.21
pycryptodome==3.15.0
pydantic==1.10.2
pydotplus==2.0.2
pydub==0.25.1
pyFUME==0.2.25
pygam==0.8.0
Pygments==2.12.0
pyGPGO==0.5.1
pygram==0.0.13b0
pymc3==3.11.5
PyNaCl==1.5.0
pyparsing==3.0.9
pyproj==3.2.1
pyreadline==2.1
pyrsistent==0.19.2
pytest==7.1.3
python-dateutil==2.8.2
python-javabridge==4.0.3
python-multipart==0.0.5
python-utils==3.3.3
python-weka-wrapper==0.3.18
python-weka-wrapper3==0.2.11
pytz==2022.1
pywin32==304
PyYAML==6.0
pyzmq==23.2.0
ray==2.1.0
requests==2.28.1
requests-oauthlib==1.3.1
rfc3986==1.5.0
rsa==4.8
scikit-fuzzy==0.4.2
scikit-learn==1.0.2
scikit-plot==0.3.7
scipy==1.7.3
seaborn==0.11.2
semver==2.13.0
simpful==2.8.0
six==1.16.0
sklearn==0.0
sklearn-weka-plugin==0.0.6
sniffio==1.3.0
starlette==0.20.4
tabulate==0.8.10
tbb==2021.7.0
tensorboard==2.9.1
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow==2.5.0
tensorflow-estimator==2.5.0
termcolor==1.1.0
Theano-PyMC==1.1.2
threadpoolctl==3.1.0
tomli==2.0.1
tornado==6.2
tqdm==4.64.1
traitlets==5.3.0
tsplib95==0.7.1
typing_extensions==4.4.0
uc-micro-py==1.0.1
urllib3==1.26.10
uvicorn==0.19.0
virtualenv==20.16.6
wcwidth==0.2.5
websockets==10.4
weka==2.0.0
Werkzeug==2.1.2
wrapt==1.12.1
xarray==0.20.2
xarray-einstats==0.2.2
xlrd==2.0.1
yarl==1.8.1
zipp==3.8.1

**Comment by fracpete** [comment]
You installed the Python 2.7 and Python 3 version of pww3:

```
python-weka-wrapper==0.3.18
python-weka-wrapper3==0.2.11
```

I recommend creating a new virtual environment:
* install pww3 and confirm that it is working, e.g. with:

    ```commandline
    python -c "import weka.core.jvm as jvm; from weka.classifiers import Classifier; jvm.start(); cls = Classifier(classname='weka.classifiers.trees.J48'); print(cls.to_commandline()); jvm.stop();"
    ```

    should output at the end:

    ```
    weka.classifiers.trees.J48 -C 0.25 -M 
    ```

* install sklweka and confirm that it is working
  
    ```commandline
    python -c "import sklweka.jvm as jvm; from sklweka.classifiers import WekaEstimator; jvm.start(); cls = WekaEstimator(classname='weka.classifiers.functions.LinearRegression'); print(cls); jvm.stop();"
    ```

    should output at the end:

    ```
    Linear Regression: No model built yet.
    ```

The virtual environment that I created on my Linux machine for the above commands, has the following libraries installed (`pip freeze`):

```
joblib==1.2.0
numpy==1.23.5
packaging==21.3
pyparsing==3.0.9
python-javabridge==4.0.3
python-weka-wrapper3==0.2.11
scikit-learn==1.1.3
scipy==1.9.3
sklearn==0.0.post1
sklearn-weka-plugin==0.0.6
threadpoolctl==3.1.0
```

**Comment by dmohsen23** [comment]
Thanks, the problem was solved. The issue was that sklweka was installed on another version of python, and I installed it on the proper python version!
